tests/test_create_estimator.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from yacs.config import CfgNode as CN
+
+from datasetinsights.estimators import FasterRCNN, create_estimator
+
+
+@pytest.fixture
+def config():
```

Comment:
This test only tries to create estimator for `FasterRCNN`, as this is the only estimator defined in the yaml.